BookService/BookService/Controllers/BooksController.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections.Generic;
+using BookService.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BookService.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BooksController : ControllerBase
```

Comment:
You should have documentation for all code that you have. Classes/method/props etx